Bottom: b0332e3d18e010cb32b7fa2f17fc6beb5afdf8ae
Top:    31b9d884790f1db362e729ffa53cfdfe0103ac29
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-01-06 21:11:46 +0000

lesstif: Use Coord type for line-width, not int


---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index c4c31ee..125a81b 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -52,7 +52,7 @@ typedef struct lesstif_gc_struct
 
   Pixel color;
   const char *colorname;
-  int width;
+  Coord width;
   EndCapStyle cap;
   char xor_set;
   char erase;
